dor/adapters/technical_metadata.py

```diff
@@ -48,8 +56,42 @@ class ImageTechnicalMetadata:
     def create(cls, file_path: Path) -> Self:
         jhove_doc = JHOVEDoc.create(
             file_path,
-            JHOVE_IMAGE_METADATA_PROPERTY
+            cls.metadata_property()
+        )
+
+        mimetype = jhove_doc.mimetype
+        if mimetype.startswith("image/"):
+            cls_ = ImageTechnicalMetadata
+        elif mimetype.startswith("text/"):
+            cls_ = TextTechnicalMetadata
+        else:
+            cls_ = cls
+
+        # set the metadata_property
+        jhove_doc.metadata_property = cls_.metadata_property()
+
+        return cls_(
+            mimetype=ImageMimetype(jhove_doc.mimetype),
+            metadata=jhove_doc.technical_metadata,
+            status=JHOVEStatus(jhove_doc.status),
+            valid=jhove_doc.valid
         )
+    
+    @classmethod
+    def metadata_property(self):
+        return "/"
+    
+    def __str__(self):
+        return ET.tostring(self.metadata, encoding="unicode")
+
+    
+@dataclass
+class ImageTechnicalMetadata(TechnicalMetadata):
+
+    
+    @classmethod
+    def via(cls, jhove_doc: 'JHOVEDoc'):
+        jhove_doc.metadata_property = JHOVE_IMAGE_METADATA_PROPERTY
```

Comment:
I don't see that we're using these `via` methods yet. Where do you see foresee needing them?

Reply:
That was an dead end approach to the `metadata_property` issue.